### DISCUSSION TITLE
AMQP.NET RabbitMQVirtual Host not working

### DISCUSSION THREAD

**Discussion by soorjithpj** [rabbitmq/rabbitmq-dotnet-client#1088] [Q&A]
Hello All,

I was working with AMQP.NET and RabbitMQ. I need to publish and receive messages to RabbitMQ queue with Virtual Host.
When I tried using the connection string, "amqp://guest:guest@localhost:5672/testvh" It is still going to the root (/) not to “/testvh”.

Can anyone help me to figure this out?

Thanks & regards,

Soorjith

**Comment by lukebakken** [comment]
Without an application to compile and run, we can't assist you.
